source/tutorial/resync-replica-set-member.txt

```diff
@@ -48,40 +48,43 @@ Procedures
 Automatically Sync a Member
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. warning:: During initial sync, :program:`mongod` will remove the
-   content of the :setting:`~storage.dbPath`.
+.. warning::
 
-This procedure relies on MongoDB's regular process for 
-:doc:`Replica Set Syncing </core/replica-set-sync>`. This will store the current data on
-the member. For an overview of MongoDB initial sync process, see the
+   During initial sync, :program:`mongod` removes the contents of the
+   :setting:`~storage.dbPath` directory.
+
+This procedure relies on MongoDB's regular process for
+:doc:`Replica Set Syncing </core/replica-set-sync>`. This stores the current
+data on the member. For an overview of MongoDB initial sync process, see the
 :doc:`Replica Set Syncing </core/replica-set-sync>` section.
 
-If the instance has no data, you can simply follow the
+Initial sync operations can impact the other members of the set and
+create additional traffic to the primary. They require another member of the
```

Comment:
just one nit -- should we clarify what "They" refer to ?

Reply:
Good idea, I'll clarify.

Reply:
Done!